docs/common_text/rasusa_task.md

```diff
@@ -1,9 +1,29 @@
-??? task "`Rasusa`: Read subsampling (optional, on by default)"
```

Comment:
please keep the description here since it describes it's optional status in another workflow